servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -95,6 +98,8 @@
 
   private final PrefixList mWhitelist;
 
+  /** The service that tries to check inodefiles with ttl set */
+  private Future<?> mInodeFileTTLService;
```

Comment:
I think the naming doesn't need the part "Inode", and should have TTLChecker, or TTLDetection or something similar.